core/src/test/java/org/elasticsearch/index/query/GeoDataGenerator.java

```diff
@@ -0,0 +1,101 @@
+/*
+ * Licensed to Elasticsearch under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.elasticsearch.index.query;
+
+import org.elasticsearch.common.geo.GeoPoint;
+import org.elasticsearch.common.geo.GeoUtils;
+import org.elasticsearch.test.ESTestCase;
+
+import static org.apache.lucene.util.LuceneTestCase.rarely;
+
+/** Helper class for generating geo related data. */
+public class GeoDataGenerator {
```

Comment:
I think most of this went away in the latest GeoBoundingBox PR - see e.g. there: https://github.com/elastic/elasticsearch/pull/11969/files#diff-c34f179cb205333bd004736b1a8a71caR46